irohad/main/application.cpp

```diff
@@ -21,12 +21,17 @@ Irohad::Irohad():
 {}
 
 void Irohad::run(){
-//  iroha::Irohad irohad;
-//  iroha::ametsuchi::StorageImpl ametsuchi;
+  auto log = logger::Logger("Irohad");
+  log.info("running");
 
+  log.info("create server's keypair");
   // TODO replace with actual public private keys
   auto seed = iroha::create_seed("some passphrase");
   auto keypair = iroha::create_keypair(seed);
+  log.info("publicKey: {}****, privateKey: {}****",
+     keypair.pubkey.to_base64().substr(0,10),
+     keypair.privkey.to_base64().substr(0,10)
```

Comment:
Logging private key potential security hole. I prefer to remove private key logging until it will be not required.